log-redaction.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Log Redaction
+summary: Learn the log redaction in TiDB components.
+---
+
+# Log Redaction
+
+When TiDB provides detailed log information, it might print sensitive data (for example, user data) in the log, which causes data security risks. To avoid such risks, each component (TiDB, TiKV, and PD) provides a configuration item that enables log redaction to shield user data values.
```

Comment:
```suggestion
When TiDB provides detailed log information, it might print sensitive data (for example, user data) in the log, which causes data security risks. To avoid such risks, each component (TiDB and TiKV) provides a configuration item that enables log redaction to shield user data values.
```